pkg/repo_node.go

```diff
@@ -95,15 +96,17 @@ func (n RepoNode) Children() ([]RepoNode, error) {
 	if err != nil {
 		return nil, fmt.Errorf("cannot parse children of node '%s': %w", n.path, err)
 	}
-	return lo.Map(children.Children, func(child nodeArrayChild, _ int) RepoNode { return n.Child(child.Name) }), nil
+	childrenWithType := lo.Filter(children.Children, func(c nodeArrayChild, _ int) bool { return c.PrimaryType != "" })
+	return lo.Map(childrenWithType, func(child nodeArrayChild, _ int) RepoNode { return n.Child(child.Name) }), nil
 }
 
 type nodeArrayChildren struct {
 	Children []nodeArrayChild `json:"__children__"`
 }
 
 type nodeArrayChild struct {
-	Name string `json:"__name__"`
+	Name        string `json:"__name__"`
+	PrimaryType string `json:"jcr:primaryType,omitempty"`
```

Comment:
root has `/is` and `/mnt` virtual nodes with only `__name__` without primary type specified